### ISSUE TITLE
This extension may have been corrupted.

### ISSUE THREAD

**Issue by mehdibahraami** [tulios/json-viewer#81]
Hi,
I need a new custom theme, and go to 
`./Library/Application Support/Google/Chrome/Default/Extensions/gbmdgpbipfallnflgajpaliibnhdgobh/0.12.0_0/themes/dark`
when i add a new theme such as other themes i don't see this theme in Extension option and when i changed other existed i give this error: 
`This extension may have been corrupted.`

**Comment by tulios** [comment]
Hi @mehdibahraami, it's not that simple to add a new theme, there are some internals that need to be configured. What do you think about the following options?

1) Switch to the default theme and use the "Custom Style" field to apply your theme, it's possible to override everything through this option. Check the Options page.

2) If the theme is free you can post the style here and I can add to the extension for everybody.

Thanks.

**Comment by mehdibahraami** [comment]
Yes, this is my theme.

**Comment by mehdibahraami** [comment]
Yes, this is my theme.

```
.cm-s-mehdi.CodeMirror {
background:#2c3e50!important;
color:#ecf0f1!important;
}

.cm-s-mehdi div.CodeMirror-selected {
background:#16a085!important;
}

.cm-s-mehdi .CodeMirror-gutters {
background:0 0!important;
border:none;
}

.cm-s-mehdi .CodeMirror-guttermarker {
color:#3498db!important;
}

.cm-s-mehdi .CodeMirror-guttermarker-subtle,.cm-s-mehdi .CodeMirror-linenumber {
color:#7f8c8d!important;
}

.cm-s-mehdi .CodeMirror-cursor {
border-color:#ccc!important;
}

.cm-s-mehdi span.cm-comment {
color:#7f8c8d!important;
}

.cm-s-mehdi span.cm-atom,.cm-s-mehdi span.cm-number {
color:#f1c40f!important;
}

.cm-s-mehdi span.cm-tag {
color:#e2777a!important;
}

.cm-s-mehdi span.cm-error {
background:0 0!important;
color:#7f8c8d!important;
}

.cm-s-mehdi .CodeMirror-matchingbracket {
color:#27ae60!important;
}

.theme-mehdi-css-check:before {
content:'loaded';
display:none;
}

.extras .icon svg {
fill:#fff;
}

.extras .icon:hover {
background-color:#333;
}

.cm-s-mehdi.CodeMirror ::selection,.cm-s-mehdi.CodeMirror ::-moz-selection {
background:rgba(73,72,62,.99);
}

.cm-s-mehdi span.cm-attribute,.cm-s-mehdi span.cm-keyword,.cm-s-mehdi span.cm-property,.cm-s-mehdi span.cm-variable,.cm-s-mehdi span.cm-variable-2 {
color:#ecf0f1!important;
}

.cm-s-mehdi span.cm-string,.cm-s-mehdi span.cm-string-2,.cm-s-mehdi span.cm-link {
color:#1abc9c!important;
}

.cm-s-mehdi span.cm-bracket,.cm-s-mehdi span.cm-def,.cm-s-mehdi .CodeMirror-activeline-background {
color:#2e383c!important;
}
```

**Comment by tulios** [comment]
<img width="382" alt="screen shot 2016-05-08 at 21 41 36" src="https://cloud.githubusercontent.com/assets/33231/15099971/accb0e62-1565-11e6-902d-d77a5023a638.png">

**Comment by tulios** [comment]
Released in version 0.13.0

**Comment by mehdibahraami** [comment]
@tulios Thanks Man.
